### PR TITLE
FC-0068 docs: Reference cleaning by edunext #2

### PR DESCRIPTION
**Pull request by Apgomeznext**
This PR modifies a series of reference documents migrated from the legacy repositories to the educators' documentation.

The principal changes are in the custom_python reference, divided into one reference, two concepts, and three how-tos.

Another important thing for reviewers is the link to edX files in the Custom Javascript reference that I couldn't erase due to their need for the proper function of that example.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Apgomeznext!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Apgomeznext!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: source/educators/references/advanced_features/planning_content_reuse.rst
Comment (jswope00) [inline source/educators/references/advanced_features/planning_content_reuse.rst:25]: "a course in an Open edX instance discussions" would be better stated "Open edX course discussions"

File: source/educators/references/advanced_features/planning_content_reuse.rst
Comment (jswope00) [inline source/educators/references/advanced_features/planning_content_reuse.rst:17]: Remove the 'a'

File: source/educators/references/advanced_features/managing_teams_via_csv.rst
Comment (jswope00) [inline source/educators/references/advanced_features/managing_teams_via_csv.rst:84]: The bolding has been removed but now we've got a stray asterisk in here. I recommend leaving this bolded and removing the extra asterisk.

File: source/educators/references/advanced_features/managing_teams_via_csv.rst
Comment (jswope00) [inline source/educators/references/advanced_features/managing_teams_via_csv.rst:44]: 'institution-email' makes me think this needs to be an .edu address or some other official email. Just change these to 'username' and 'email'

File: source/educators/references/advanced_features/experiment_group_configurations.rst
Comment (jswope00) [inline source/educators/references/advanced_features/experiment_group_configurations.rst:1]: "instance platform" is redundant, IMO. Choose one or the other and be consistent with it throughout the doc.

File: source/educators/concepts/exercise_tools/award_partial_credit.rst
Comment (jswope00) [inline source/educators/concepts/exercise_tools/award_partial_credit.rst:1]: I would argue this is a how-to. The verb here is "award", which generally is a giveaway of a how-to doc.

File: source/educators/concepts/exercise_tools/script_tag_format.rst
Comment (jswope00) [inline source/educators/concepts/exercise_tools/script_tag_format.rst:1]: I would argue this is a reference. It is a specific details about a format. This kind of list, table, or other details is more reference than a concept which usually covers bigger-picture ideas.

File: source/educators/references/advanced_features/experiment_group_configurations.rst
Comment (sarina) [reply source/educators/references/advanced_features/experiment_group_configurations.rst:1]: I use "instance" and "platform" interchangeably, but _never_ at the same time. Use either "Open edX instance" OR "Open edX platform" - I think sometimes "instance" is more correct while "platform" can often refer to the codebase itself as a platform (as opposed to a local instance)

File: source/educators/references/advanced_features/managing_teams_via_csv.rst
Comment (sarina) [inline source/educators/references/advanced_features/managing_teams_via_csv.rst:44]:
```suggestion
(which, in priority order, is the ``student-key``, ``username``, or ``email``),
```

File: source/educators/references/advanced_features/managing_teams_via_csv.rst
Comment (sarina) [inline source/educators/references/advanced_features/managing_teams_via_csv.rst:45]:
```suggestion
``enrollment mode`` (one of ``audit``, ``verified``, or ``masters``), and the ``team`` assignment for the matching
```

File: source/educators/references/advanced_features/teams_configuration_options.rst
Comment (sarina) [inline source/educators/references/advanced_features/teams_configuration_options.rst:35]:
```suggestion
   running, if learners might have already joined teams within topics. If
```

**Review by sarina — CHANGES_REQUESTED** [review]

**Comment by Apgomeznext** [comment]
@sarina @jswope00 The requested changes have been made. Let me know if there is any else to do with these files.

**Review by sarina — APPROVED** [review]

**Review by jswope00 — APPROVED** [review]

**Pull request merged**
